service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -55,10 +51,6 @@ public boolean isSendLAEmailsToSolicitorEnabled(String localAuthorityName) {
             createLDUser(Map.of(LOCAL_AUTHORITY_NAME_KEY, LDValue.of(localAuthorityName))), false);
     }
 
-    public boolean isUploadDraftCMOEnabled() {
```

Comment:
not really belongs to this ticket, but should be ok

Reply:
will keep it